### PR TITLE
[16.0][ADD] account_move_sent_usability

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: account_move_sent_usability/views/account_move.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <field name="name">account.move.filter</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter" />
+        <field name="arch" type="xml">
+
+            <xpath expr="//filter[@name='open']/preceding::separator" position="before">
+                <separator />
+                <filter
+                    name="unsent"
+                    string="Unsent"
```
Comment (IT-Ideas) [inline account_move_sent_usability/views/account_move.xml]: Unsent sounds odd to me. Shouldn't we rather use this?
```suggestion
                    string="Not Sent"
```

**Review by IT-Ideas — APPROVED** [review]
Thanks!

File: account_move_sent_usability/views/account_move.xml (hunk 1 above)
Comment (jbaudoux) [reply account_move_sent_usability/views/account_move.xml]: Indeed, wording is not correct

**Review by sbidoul — APPROVED** [review]

File: account_move_sent_usability/views/account_move.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter" />
+        <field name="arch" type="xml">
+
+            <xpath expr="//filter[@name='open']/preceding::separator" position="before">
+                <separator />
+                <filter
+                    name="unsent"
+                    string="Not Sent"
+                    domain="[('is_move_sent', '=', False)]"
```
Comment (sbidoul) [inline account_move_sent_usability/views/account_move.xml:17]:
```suggestion
                    domain="[('is_move_sent', '=', False), ('state', '!=', 'cancel')]"
```

I suggest to not show cancelled invoice in this "not sent" filter, as this unnecessarily pollutes the result with invoices that will never be sent.

Comment (sbidoul) [reply account_move_sent_usability/views/account_move.xml:17]: Or not. It's easy enough to click on the Draft and Posted filters in addition to Not Sent

File: account_move_sent_usability/views/account_move.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015-2018 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
```
Comment (jbaudoux) [inline account_move_sent_usability/views/account_move.xml]:
```suggestion
<!-- Copyright 2015 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
```

File: account_move_sent_usability/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 ACSONE SA/NV
```
Comment (jbaudoux) [inline account_move_sent_usability/__manifest__.py:2]:
```suggestion
# Copyright 2015 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
# Copyright 2023 ACSONE SA/NV
```

File: account_move_sent_usability/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Move Sent Usability",
+    "summary": """
+        Allows to filter moves on 'is_move_sent' and to see the value of the field in form""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
```
Comment (jbaudoux) [inline account_move_sent_usability/__manifest__.py]:
```suggestion
    "author": "BCIM,ACSONE SA/NV,Odoo Community Association (OCA)",
```

**Review by jbaudoux — APPROVED** [review]

File: account_move_sent_usability/views/account_move.xml (hunk 2 above)
Comment (jbaudoux) [reply account_move_sent_usability/views/account_move.xml:17]: In fact, it was like that in previous versions. The purpose of the filter was to select the invoices to sent and launch the mass sending action wizard. It was convenient to exclude the canceled invoices. Now that you can select all lines in the list view without showing them all, it is fine for me to remove this from the filter.

**Pull request merged**